src/m365/teams/commands/chat/chat-message-send.ts

```diff
@@ -0,0 +1,244 @@
+      {
+        option: '-m, --message <message>'
+      }
+    ];
+
+    const parentOptions: CommandOption[] = super.options();
+    return options.concat(parentOptions);
+  }
+
+  public validate(args: CommandArgs): boolean | string {    
+    if (!args.options.chatId && !args.options.userEmails && !args.options.chatName) {
+      return 'Specify chatId or userEmails or chatName, one is required.';
+    }
+
+    let nrOfMutuallyExclusiveOptionsInUse = 0;
+    if (args.options.chatId) { nrOfMutuallyExclusiveOptionsInUse++; }
+    if (args.options.userEmails) { nrOfMutuallyExclusiveOptionsInUse++; }
+    if (args.options.chatName) { nrOfMutuallyExclusiveOptionsInUse++; }
+
+    if (nrOfMutuallyExclusiveOptionsInUse > 1) {
+      return 'Specify either chatId or userEmails or chatName, but not multiple.';
+    }
+
+    if (!args.options.message) {
+      return 'Specify a message to send.';
+    }
+
+    if (args.options.chatId && !Utils.isValidTeamsChatId(args.options.chatId)) {
+      return `${args.options.chatId} is not a valid Teams ChatId.`;
+    }
+
+    if (args.options.userEmails) {
+      const userEmails = args.options.userEmails.toLowerCase().replace(/\s/g,"").split(",").filter(e => e && e !== "");
+      if (!userEmails || userEmails.length === 0 || userEmails.some(e => !Utils.isValidUserPrincipalName(e))) {
+        return `${args.options.userEmails} contains one or more invalid email addresses.`;
+      }
+    }
+
+    return true;
+  }
+  
+  private async ensureChatIdByUserEmails(userEmailsOption: string): Promise<string | undefined> {    
+    const userEmails = userEmailsOption.toLowerCase().replace(/\s/g,"").split(",").filter(e => e && e !== "");
+    const currentUserEmail = Utils.getUserNameFromAccessToken(Auth.service.accessTokens[this.resource].accessToken).toLowerCase();    
+    const existingChats = await this.findExistingGroupChatsByMembers([currentUserEmail, ...userEmails]);
+
+    if (existingChats && existingChats.length > 0) {
+      if (existingChats.length > 1) {
+        const disambiguationText = existingChats.map(c => {
+          return `- ${c.id}${c.topic && " - "}${c.topic} - ${c.createdDateTime && new Date(c.createdDateTime).toLocaleString()}`;
+        }).join(os.EOL);
+
+        throw new Error(`Multiple chat conversations with this topic found. Please disambiguate:${os.EOL}${disambiguationText}`);
+      } 
+      else {
+        return existingChats[0].id;
+      }
+    }
+
+    const chat = await this.createConversation([currentUserEmail, ...userEmails]);
+    return chat.id;
+  }
+
+  private async getChatIdByName(chatName: string): Promise<string> {    
+    const existingChats = await this.findExistingGroupChatsByTopic(chatName);
+        
+    if (!existingChats || existingChats.length === 0) {          
+      throw new Error("No chat conversation was found with this name.");
+    }
+    else if (existingChats.length === 1) {
+      return existingChats[0].id as string;
+    }
+    else {
```

Comment:
Since we had a `return` statement in the previous block, we can drop this `else` clause and just include the code avoiding unnecessary nesting

Reply:
See my comment above.